x-pack/plugins/ml/server/routes/annotations.js

```diff
@@ -31,7 +31,8 @@ export function annotationRoutes(server, commonRouteConfig) {
     handler(request) {
       const callWithRequest = callWithRequestFactory(server, request);
       const { indexAnnotation } = annotationServiceProvider(callWithRequest);
-      return indexAnnotation(request.payload)
+      const username = _.get(request, 'auth.credentials.username', '<user unknown>');
```

Comment:
`<user unknown>` should perhaps go into the `annotations.ts` constants file.

Reply:
👍 Done in 2597d81.